### ISSUE TITLE
提交 Bug

### ISSUE THREAD

**Issue by xxxDo1** [dnGrep/dnGrep#1199]
来回频繁选择点击, 会有明显卡顿, 无法点击选中, 有时候程序会卡死, 无法关闭程序

![Video_2024-09-25_03·41·03](https://github.com/user-attachments/assets/b0aa8e28-dd25-4440-9dfe-581fe6a85ff6)

**Comment by doug24** [comment]
I'm not sure I see any issue. I have not been able to reproduce this, but maybe just because I am not clicking as quickly as you are. But what is the point of changing the selection as rapidly as possible? 

Interesting to note that you can also change the selection using the up and down arrow keys.  That moves the selection very quickly and I don't see any delay.

**Comment by xxxDo1** [comment]
快速的更改选择很有意义啊
我想要快速查看内容, 双击调用外部的查看器查看

![Video_2024-09-26_04·22·08](https://github.com/user-attachments/assets/91373e23-f369-4e5d-a426-ed8ac79fd0e1)

**Comment by xxxDo1** [comment]
我找到Bug了, 看下面git图片, 我鼠标双击的位置

![Video_2024-09-26_04·47·43](https://github.com/user-attachments/assets/61691db0-dfa4-411b-8322-b47378f495a1)

**Comment by doug24** [comment]
Yes, there is a small dead space between rows where neither row is selected or custom editor is not called up.
I don't think this is a serious enough problem to require a change.

**Comment by xxxDo1** [comment]
这个死区范围有大了, 每10次点击就能击中2次
这会造成卡顿延迟的感觉

**Comment by xxxdo1** [comment]
This dead zone is more than 2 pixels, at least 4 pixels

**Comment by doug24** [comment]
In [v4.2.95](https://github.com/dnGrep/dnGrep/releases/tag/v4.2.95.0)
